src/ResourceManager/DevTestLabs/Commands.DevTestLabs.Test/ScenarioTests/DevTestLabsTestFixture.ps1

```diff
@@ -0,0 +1 @@
+﻿
```

Comment:
This file is required because of DevTestLabsTestFixture.cs. I preferred put the two scripts in common.ps1 that's also loaded. That's why the file is empty.